src/Migrations/Version20200409125719.php

```diff
@@ -0,0 +1,42 @@
+<?php
+
+declare(strict_types=1);
+
+namespace DoctrineMigrations;
+
+use Doctrine\DBAL\Schema\Schema;
+use Doctrine\Migrations\AbstractMigration;
+
+final class Version20200409125719 extends AbstractMigration
+{
+    public function getDescription(): string
+    {
+        return 'Add search indexes on user and commissionable_asset tables.';
+    }
+
+    public function up(Schema $schema): void
+    {
+        $this->abortIf('postgresql' !== $this->connection->getDatabasePlatform()->getName(), 'Migration can only be executed safely on \'postgresql\'.');
+
+        $this->addSql('CREATE INDEX user_first_name_lower_idx ON users (LOWER(first_name))');
+        $this->addSql('CREATE INDEX user_last_name_lower_idx ON users (LOWER(last_name))');
+        $this->addSql('CREATE INDEX user_email_address_lower_idx ON users (LOWER(email_address))');
+        $this->addSql('CREATE INDEX user_identification_number_lower_idx ON users (LOWER(identification_number))');
+        $this->addSql('CREATE INDEX commissionable_asset_name_lower_idx ON commissionable_asset (LOWER(name))');
+        $this->addSql('CREATE INDEX commissionable_asset_contact_lower_idx ON commissionable_asset (LOWER(contact))');
+        $this->addSql('CREATE INDEX organization_name_lower_idx ON organization (LOWER(name))');
```

Comment:
We don't the scale that we are targeting with this project, but for high volume these indexes could cause degraded performances on the users table. Basically for every upserts we will have 4 upserts in addition ... 

My knowledge is limited on PostgreSQL, but instead i would try something like : 


```suggestion
        $this->addSql('CREATE INDEX user_searchON people ((LOWER(first_name)|| '  ' || LOWER(last_name) || '  ' || LOWER(email_address)  || '  ' || LOWER(identification_number) ));');
```

Reply:
That would be a great improvement, could you open a PR to fix it in a new migration please (to ensure no BC break)?